cluster/saltbase/salt/kubelet/default

```diff
@@ -3,23 +3,10 @@
   {% set daemon_args = "" -%}
 {% endif -%}
 
-{% if grains.api_servers is defined -%}
-  {% set api_servers = "--api-servers=https://" + grains.api_servers -%}
-{% elif grains.apiservers is defined -%} # TODO(remove after 0.16.0): Deprecated form
-  {% set api_servers = "--api-servers=https://" + grains.apiservers -%}
-{% elif grains['roles'][0] == 'kubernetes-master' -%}
```

Comment:
This should work on GCE/GKE but there may be other folks using this salt config that were depending  on these two else cases.